### ISSUE TITLE
Web API: inconsistencies on api-key resource

### ISSUE THREAD

**Issue by mguidon** [ITISFoundation/osparc-simcore#4959]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which deploy/s?

other (e.g. local)

### Current Behavior

1. Code is required but does no make sense
![image](https://github.com/ITISFoundation/osparc-simcore/assets/33161876/cec57b92-8200-4567-846d-fcbb58841a4d)

2. Expiration seems useless here:
 
![image](https://github.com/ITISFoundation/osparc-simcore/assets/33161876/d263920b-e515-4625-a8ac-d93cdf06855c)

```[tasklist]
### Event Horizon
- [ ] https://github.com/ITISFoundation/osparc-simcore/pull/6843
```

**Comment by pcrespov** [comment]
- Related with https://github.com/ITISFoundation/osparc-simcore/issues/5022

**Comment by pcrespov** [comment]
- review API in comparison with e.g. `folders` resources
- fix delete
- update response and request body with proper models.

**Comment by giancarloromeo** [comment]
Solved in #6843
